crates/query-engine/translation/tests/snapshots/tests__it_select_where_in_variable.snap

```diff
@@ -9,7 +9,7 @@ FROM
     SELECT
       row_to_json("%5_universe") AS "universe"
     FROM
-      jsonb_to_recordset($1) AS "%0_%variables_table"("%variable_order" int, "%variables" jsonb)
+      jsonb_to_recordset($1) AS "%0_%variables_table"("%variable_order" "int4", "%variables" "jsonb")
```

Comment:
Most of the changes in this PR are knock-on effects on test snapshots from type name identifiers now being quoted.